### ISSUE TITLE
cannot see the dates

### ISSUE THREAD

**Issue by 21sorenkierkegaard** [fiftythreecorp/LeagueOfLegendsAccountChecker#14]
Hey!
I cannot see the dates when a skin or a champion was bought.
In the previous versions I was able to see that,but since you added proxy and made a black theme I cannot do it anymore.
Is there any solution to this?
I just want to see the dates so I can try to verify verified accounts which are inactive.
I also tried to contact you on telegram but you didn't responds.
Please someone help me,is there any way to use the previous version if it is not patched?

I just need to verify an account so I can start playing.

**Comment by razvandada33** [comment]
Me too! All we have to do it's just to wait,but i'm 100% sure it will not happend becuase you can get your ''account'' banned because of riot new system soooo...:/ I got my account suspended because of this lol account checker i am not 100% but i remember i was on this lol checker when i was loged on my account and a window pop up saying "Your account is suspended'' when i was loged in account :/ right now i send a ticket and i am waiting to get unsuspended if i am lucky i will go unsuspended if not idk.I think will appear a version for old patcher to see when the account was created and more.But not now because as i saided before,riot has a new system..is bullshit more than the old one.

**Comment by 21sorenkierkegaard** [comment]
Hi!
They can't ban your account unless you didnt't use proxies properly.
Because all you're doing by checking those accounts is to log into them
from different zones,but if you're not using proxies you will probably log
in over and over from the same IP adress,which leads to it getting a ban.

**Comment by razvandada33** [comment]
yeah that's make sense now,but now how can i get my account unsuspended?
